### DISCUSSION TITLE
CORS preflight issue

### DISCUSSION THREAD

**Discussion by RamaHerbin** [umami-software/umami#2150] [Q&A]
Hello everyone, 
I have an issue when I post login credential to the /api/auth/login endpoint.

I request from my front-end on localhost:3000 to umami on localhost:8080

The request works fine in Postman.
What am I doing wrong ?

```javascript
  fetch("http://localhost:8080/api/auth/login", {
    method: "POST",
    headers: {
      "Content-Type": "application/json",
      'Accept': '*/*',
      "Access-Control-Allow-Headers": "*",
      "Access-Control-Allow-Methods": "*",
      "Access-Control-Allow-Origin": "*",
    },
    body: JSON.stringify(formState),
  })
```

![Screenshot 2023-04-28 at 13 59 16](https://user-images.githubusercontent.com/41597427/235141917-a433db80-3ed3-4f73-9377-385fc62276dd.jpg)

_Originally posted by @RamaHerbin in https://github.com/umami-software/umami/issues/1952#issuecomment-1527465944_

**Comment by Dumbyarnik** [comment]
Hi, did you find a solution for that?

**Reply by RamaHerbin** [reply]
I didn't :/
